commit 832819d11d4a21780958a3ee3d1d3a7a4a469d92
Author: Arnd Bergmann <arnd@arndb.de>
Date:   Tue Oct 2 17:12:43 2018 -0400

    media: i2c: TDA1997x: select CONFIG_HDMI
    
    [ Upstream commit 79e89e36dc8a47ef965a35b484d737a5227feed1 ]
    
    Without CONFIG_HDMI, we get a link error for this driver:
    
    drivers/media/i2c/tda1997x.o: In function `tda1997x_parse_infoframe':
    tda1997x.c:(.text+0x2195): undefined reference to `hdmi_infoframe_unpack'
    tda1997x.c:(.text+0x21b6): undefined reference to `hdmi_infoframe_log'
    drivers/media/i2c/tda1997x.o: In function `tda1997x_log_infoframe':
    tda1997x.c:(.text.unlikely+0x13d3): undefined reference to `hdmi_infoframe_unpack'
    tda1997x.c:(.text.unlikely+0x1426): undefined reference to `hdmi_infoframe_log'
    
    All other drivers in this directory that use HDMI select CONFIG_HDMI,
    so do the same here:
    
    Fixes: 9ac0038db9a7 ("media: i2c: Add TDA1997x HDMI receiver driver")
    
    Signed-off-by: Arnd Bergmann <arnd@arndb.de>
    Signed-off-by: Hans Verkuil <hverkuil-cisco@xs4all.nl>
    Signed-off-by: Mauro Carvalho Chehab <mchehab+samsung@kernel.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/drivers/media/i2c/Kconfig b/drivers/media/i2c/Kconfig
index 82af97430e5b..041a777dfdee 100644
--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -61,6 +61,7 @@ config VIDEO_TDA1997X
 	depends on VIDEO_V4L2 && I2C && VIDEO_V4L2_SUBDEV_API
 	depends on SND_SOC
 	select SND_PCM
+	select HDMI
 	---help---
 	  V4L2 subdevice driver for the NXP TDA1997x HDMI receivers.
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1087_832819d11d4a21780958a3ee3d1d3a7a4a469d92.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1087_832819d11d4a21780958a3ee3d1d3a7a4a469d92.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
